nistats/first_level_model.py

```diff
@@ -315,6 +328,8 @@ def __init__(self, t_r=None, slice_time_ref=0., hrf_model='glover',
         self.labels_ = None
         self.results_ = None
         self.subject_label = subject_label
+        self.low_pass = low_pass
+        self.high_pass = high_pass
```

Comment:
But then, you need to test that the new behavior is enforced. and you should remove the 'period_cut' argument as it is redundant with high_pass.

Reply:
We could not write tests, her computer had no space to install nosetests and it's dependencies.